template/cordova/lib/log.js

```diff
@@ -0,0 +1,227 @@
+    }).then(function(state) {
+        adminLogInitialState = state;
+        return enableLogging('Microsoft-Windows-AppHost/Admin');
+    }).then(function () {
+        return enableLogging('Microsoft-Windows-AppHost/ApplicationTracing');
+    }).then(function () {
+        console.log('Now printing logs. To stop, please press Ctrl+C once.');
+        startLogging('Microsoft-Windows-AppHost/Admin');
+        startLogging('Microsoft-Windows-AppHost/ApplicationTracing');
+    }).catch(function (error) {
+        console.error(error);
+    });
+
+    // Disable logs before exiting
+    process.on('exit', function () {
+        if ((appTracingLogInitialState !== null) && !appTracingLogInitialState) {
+            disableLogging('Microsoft-Windows-AppHost/ApplicationTracing');
+        }
+        if ((adminLogInitialState !== null) && !adminLogInitialState) {
+            disableLogging('Microsoft-Windows-AppHost/Admin');
+        }
+    });
+
+    // Catch Ctrl+C event and exit gracefully
+    process.on('SIGINT', function () {
+        process.exit(2);
+    });
+
+    // Catch uncaught exceptions, print trace, then exit gracefully
+    process.on('uncaughtException', function(e) {
+        console.log(e.stack);
+        process.exit(99);
+    });
+};
+
+function startLogging(channel) {
+    var startTime = new Date().toISOString();
+    setInterval(function() {
+        var command = 'wevtutil qe ' + channel + ' /q:"*[System [(TimeCreated [@SystemTime>\'' + startTime + '\'])]]" /e:root';
+        cp.exec(command, function (error, stdout, stderr) {
+            if (error) {
+                throw new Error('Failed to run wevtutil command. ' + error);
+            } else {
+                parseEvents(stdout).forEach(function (evt) {
+                    startTime = evt.timeCreated;
+                    console.log(stringifyEvent(evt));
+                });
+            }
+        });
+    }, 1000);
+}
+
+module.exports.help = function() {
+    console.log('Usage: ' + path.relative(process.cwd(), path.join(ROOT, 'cordova', 'log')));
+    console.log('Continuously prints the windows logs output to the command line.');
+    process.exit(0);
+};
+
+function getElementValue(et, element, attribute) {
+    var result;
+
+    var found = et.findall(element);
+    if (found.length > 0) {
+        if (!!attribute) {
+            result = found[0].get(attribute);
+        } else {
+            result = found[0].text; 
+        }
+    }
+
+    return result;
+}
+
+function parseEvents(output) {
+    var etree = et.parse(output);
+    var events = etree.getroot().findall('./Event');
+    var results = [];
+
+    events.forEach(function (event) {
+        // Get only unhandled exceptions from Admin log
+        if (getElementValue(event, './System/Channel') === 'Microsoft-Windows-AppHost/Admin' &&
+            typeof getElementValue(event, './UserData/WWAUnhandledApplicationException') === 'undefined') {
```

Comment:
I tested and my app which has an unhandledexception does not end up emitting an event in the log - why?